cdc/processor/sourcemanager/engine/pebble/event_sorter.go

```diff
@@ -99,10 +99,11 @@ func (s *EventSorter) AddTable(tableID model.TableID) {
 	s.mu.Lock()
 	if _, exists := s.tables[tableID]; exists {
 		s.mu.Unlock()
-		log.Panic("add an exist table",
```

Comment:
When does it happen?

Reply:
When etcd errors happen, the capture will suicide and call `processorManger.AsyncClose`, but It seems still can `tick`. So it will add table again when the engine hasn't been dropped.